Community PR - Dune Dashboard (2a) (#5976)
w3f/polkadot-wiki: docs/general/dune-analytics/parachain-dashboards/polimec-dashboards.md

---
id: polimec-dashboards
title: Polimec Dashboards
sidebar_label: Polimec
description:

>   Polimec offers a decentralized, transparent, and compliant method for fundraising, ensuring that
>   stakeholder incentives are aligned both during and after the fundraising process.

Reviewer: ```suggestion
  Polimec offers a decentralized, transparent, and compliant method for fundraising.
```

keywords: [polkadot, dashboard, dune, polimec, PLMC, funding, staking]
slug: ../polimec-dashboards
---

# Polimec Dashboards

## Overview

Polimec offers a decentralized, transparent, and compliant method for fundraising, ensuring that
stakeholder incentives are aligned both during and after the fundraising process.

## Featured Dashboards on Dune

> Here you'll find a variety of dashboards that help visualize data from the Moonbeam parachain:

Reviewer: ```suggestion
Here you will find a variety of dashboards that help visualize data from the Moonbeam parachain:
```

- [Polimec](https://dune.com/substrate/polimec): A comprehensive analysis of NeuroWeb, including:
  DKG, knowledge asset, asset, and XCM analysis.

## Key Tables

Data from the polimec parachain is organized into several key tables:

- `polimec.balances`
- `polimec.blocks`
- `polimec.calls`
- `polimec.events`,
- `polimec.extrinsics`
- `polimec.transfers`

## Useful Queries

> Currently, there are no specific useful queries provided. Please check back later as this section
> will be updated with materialized queries for Neuroweb.

Reviewer: ```suggestion
Currently, there are no specific queries provided.
```
